core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -132,7 +132,7 @@ public static Snapshot oldestAncestor(Table table) {
    * @param lookup lookup function from snapshot ID to snapshot
    * @return null if there is no current snapshot in the table, else the oldest Snapshot.
    */
-  public static Snapshot oldestAncestor(long snapshotId, Function<Long, Snapshot> lookup) {
+  public static Snapshot oldestAncestorOf(long snapshotId, Function<Long, Snapshot> lookup) {
```

Comment:
Didn't this exist before the other commit? If this has been released, I doubt it is a good idea to rename it just to add "Of".

Reply:
this was added by the other PR #4744 (incremental append scan impl) that just merged. It is not released and should be safe to rename

Reply:
It was just added and hasn't been released. All other methods in this class follow this name pattern.